windows/vhba_hot_add_remove/create_new_file_read_write.yml

```diff
@@ -0,0 +1,103 @@
+- name: "Check file created on new disk in guest OS"
+  ansible.builtin.assert:
+    that:
+      - "{{ win_check_file_exist_result }}"
+    success_msg: "New created file found in guest OS"
+    fail_msg: "New created file not found in guest OS: {{ write_test_file_path_win }}"
+
+- include_tasks: ../utils/win_execute_cmd.yml
+  vars:
+    win_powershell_cmd: "type {{ write_test_file_path_win }}"
+- name: "Save the result of getting new file content"
+  ansible.builtin.set_fact:
+    get_file_content_result1: "{{ win_powershell_cmd_output }}"
+- name: "Check file content"
+  ansible.builtin.assert:
+    that:
+      - get_file_content_result1.stdout is defined
+      - get_file_content_result1.stdout
+      - "test_line in get_file_content_result1.stdout"
+    fail_msg: "The file content is '{{ get_file_content_result1.stdout }}', which should contain: '{{ test_line }}'"
+
+# Check new created file still exist after guest OS restart
+- include_tasks: ../utils/win_shutdown_restart.yml
+  vars:
+    set_win_power_state: "restart"
+- include_tasks: ../utils/win_execute_cmd.yml
+  vars:
+    win_powershell_cmd: "type {{ write_test_file_path_win }}"
+    win_execute_cmd_ignore_error: true
+- name: "Save the result of getting new file content after guest restart"
+  ansible.builtin.set_fact:
+    get_file_content_result2: "{{ win_powershell_cmd_output }}"
+
+# If get new file content failed check disk status firstly
+- name: "Get new disk online status after guest restart"
+  block:
+    - include_tasks: ../utils/win_get_set_disk_online.yml
+      vars:
+        win_online_disk_uid: "{{ win_disk_unique_id }}"
+    - name: "Set fact of new disk online status"
+      ansible.builtin.set_fact:
+        new_disk_is_online: "{{ win_is_disk_online }}"
+  when:
+    - get_file_content_result2.rc is defined
+    - get_file_content_result2.rc != 0
+
+- name: "Handle get new file content failure"
+  block:
+    - name: "Known issue - workaround of hot adding disk to new LSILogicSAS controller in Windows Server"
+      ansible.builtin.debug:
+        msg:
+          - "After Windows Server guest restart the new added disk to the new LSILogicSAS controller will be offline. Ignore this known issue."
+          - "Online this disk as a workaround for hot adding disk to the new LSILogicSAS controller test."
+      tags:
+        - known_issue
+    - include_tasks: ../utils/win_get_set_disk_online.yml
+      vars:
+        win_online_disk_uid: "{{ win_disk_unique_id }}"
+        win_online_disk_ops: 'set'
+    - include_tasks: ../utils/win_execute_cmd.yml
+      vars:
+        win_powershell_cmd: "type {{ write_test_file_path_win }}"
+        win_execute_cmd_ignore_error: true
+    - name: "Save the result of getting new file content after online disk"
+      ansible.builtin.set_fact:
+        get_file_content_result2: "{{ win_powershell_cmd_output }}"
+  when:
+    - get_file_content_result2.rc is defined
```

Comment:
get_file_content_result2.rc checking can be removed here, as new_disk_is_online is defined means it is set when get_file_content_result2.rc is defined and get_file_content_result2.rc != 0.